src/plugins/controls/public/control_types/range_slider/range_slider_popover.tsx

```diff
@@ -52,6 +56,13 @@ export const RangeSliderPopover: FC<Props> = ({
   let helpText = '';
 
   const hasAvailableRange = min !== '' && max !== '';
+
+  if (!hasAvailableRange) {
+    helpText = RangeSliderStrings.popover.getNoAvailableDataHelpText();
+  } else if (isInvalid) {
+    helpText = RangeSliderStrings.popover.getNoDataHelpText();
```

Comment:
I was also able to get in to a "No data" state, with validation on, but without any error messages, via the following:
1. Create two range slider controls - one on `products.price` and the other on `products.discount_percentage`
2. Set the `products.price` range to `300 -> 500`
3. Set the `products.discount_percentage` range to `2 -> 20`
4. Notice that you end up with no data but no error messages and/or ignored filters - so it seems like the validation isn't working with chaining properly?

Video:

https://user-images.githubusercontent.com/8698078/167179094-76545453-2b84-42ce-b9c6-6cad2e17fbd7.mp4

Reply:
This was actually an misunderstanding of how things work with multi-fields and not actually a chaining issue - I filed a [related issue](https://github.com/elastic/kibana/issues/132367) to discuss how we could make this behaviour clearer.